### PR TITLE
Add two new alert runbook docs

### PR DESCRIPTION
**Pull request by aruniiird**

### CODE REVIEW

File: alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# MDSCpuUsageHighNeedsHorizontalScaling
```
Comment (parth-gr) [inline alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md:1]: We should remove the alert 
https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/CephMdsCpuUsageHigh.md,
As we are adding new two alerts

File: alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+the alert is fired if the used CPU is 67% of allocated CPU for 6 hours.
+If this is the case take the steps mentioned in mitigation.
+
+## Mitigation
+
+We need to either increase the allocated CPU or run multiple active MDS. The
+below command describes how to set the number of allocated CPU for MDS server.
+
+```bash
+oc patch -n openshift-storage storagecluster ocs-storagecluster \
```
Comment (parth-gr) [inline alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md:32]: why we are increasing cpu in horizontal scaling

File: alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## Impact
+
+MDS serves filesystem metadata. The MDS is crucial for any file creation,
+rename, deletion and update operations.
+MDS is by default is allocated 2 or 3 CPUS.
+It is okay as long as metadata operations are not too many.
+When the metadata operation load increases enough to trigger this alert,
+it means the default CPU allocation is unable to cope with load and we need to
+increase the CPU allocation or run multiple active metadata servers.
```
Comment (parth-gr) [inline alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md:16]: Remove the `or`

**Review by parth-gr — CHANGES_REQUESTED** [review]
We should remove the alert 
https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/CephMdsCpuUsageHigh.md,
As we are adding new 2 alerts

File: alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md (hunk 1 above)
Comment (aruniiird) [reply alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md:1]: @parth-gr ,
I understand your point, but unfortunately all our alerts (in [prometheus-ocs-rules.yaml](https://github.com/red-hat-storage/ocs-operator/blob/main/metrics/deploy/prometheus-ocs-rules.yaml) file) is pointing to this `master` repo files.
So if we remove any existing alerts' runbook entry (from this repo), we won't be able to access it from the `prometheus-ocs-rules.yaml` file (from the other repo) for all the older releases (like 4.17 or 4.16 etc).

That means we can add new alerts, but removal can only be done when the last older release (which points to this runbook) goes out of commision (obselete, no more supported).

File: alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md (hunk 3 above)
Comment (aruniiird) [reply alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md]: Updated the doc

File: alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md (hunk 2 above)
Comment (aruniiird) [reply alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md]: You are right, updated the doc

File: alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md (hunk 1 above)
Comment (parth-gr) [reply alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md:1]: okay, hope that won't be visible now for 4.18

File: alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsVerticalScaling.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# MDSCpuUsageHighNeedsVerticalScaling
+
+## Meaning
+
+Ceph MDS cpu usage for the MDS daemon has exceeded the threshold for adequate
```
Comment (jmolmo) [inline alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsVerticalScaling.md:5]: Type: "cpu" is used in capital letters in all other places

**Review by jmolmo — APPROVED** [review]
just fix the "cpu" not in capital letters

File: alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsVerticalScaling.md (hunk 4 above)
Comment (aruniiird) [reply alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsVerticalScaling.md]: Fixed... Thanks

**Pull request merged**